File: ws/fuentes/U3/WS28Taxes/Taxes/src/test/java/ec/edu/espe/taxes/controller/OperationTest.java
package ec.edu.espe.taxes.controller;

import org.junit.jupiter.api.Test;
import static org.junit.jupiter.api.Assertions.*;

/**
 *
 * @author LABS-ESPE
 */
public class OperationTest {
    
    public OperationTest() {
    }

    @Test
    public void testAdd() {
        System.out.println("add");
        float addend1 = 1.2F;
        float addend2 = 2.4F;
        float expResult = 3.6F;
        float result = Operation.add(addend1, addend2);
        assertEquals(expResult, result, 0);
    }
    
}

File: hw/fuentes/U3/HW16UNITTEST/TestPy/Taxes/ec/edu/espe/taxes/Test/OperationTest.py
import unittest
from controller.Operation import Operation

class TestOperation(unittest.TestCase):
    def test_add(self):
        result = Operation.add(1.2, 2.4)
        self.assertAlmostEqual(result, 3.6)

if __name__ == '__main__':
    unittest.main()
